flexmeasures/ui/views/logged_in_user.py

```diff
@@ -18,11 +25,31 @@ def logged_in_user_view():
     """
     account_roles = get_account_roles(current_user.account_id)
     account_role_names = [account_role.name for account_role in account_roles]
+    account = db.session.execute(
+        select(Account).filter_by(id=current_user.account_id)
+    ).scalar()
+
+    try:
```

Comment:
Maybe write a little shorter as:

```
user_can_view_account_auditlog = True
try:
    check_access(AuditLog.account_table_acl(account), "read")
except (Forbidden, Unauthorized):
    user_can_view_account_auditlog = False
```

Note that `Unauthorized` can also be raised!

Reply:
You're right! I reviewed check_access again and spotted the Unauthorized handling. Thanks for the heads-up—I’ll push a commit to address this.